README.md

```diff
@@ -1,2 +1,25 @@
 # Seine-HelloWorld
 The testing repo for Seine Bot
+
+
+# Assignment 1 (5/19/24) Test Event Types for Seine AI
+
+1. Fork this repository. Create/Edit a file of your choice.
+2. Create a PR with your commmited changes
+3. Test out the following event types:
+ - add PR Request review
+ - specific line comment review 
+
+Demos below have more details:
+
+*add PR request review should automatically run when you make a PR*
+
+after you push your commits. on github (or remotely) create a new PR with your commited changes. Once you create the PR, the bot should automatically run.
+![add new pull request image](image.png)
+
+now the bot should be activated on Github actions
+![action page](image-2.png)
+![bot is processing the review](image-1.png)
+
+yay! the bot has given me a review on my PR. review the pr and make sure it make sense with your code. 
```

Comment:
This comment is asking the author of the pull request to review the text for grammar and tone. It is likely that the pull request contains changes to documentation or comments in the code, and the reviewer wants to ensure that the writing is clear and professional. The reviewer is addressing the author by their GitHub username, @SeineSailor, to get their attention and request a specific action.

To respond to this comment, the author could review the text and make any necessary changes to improve the grammar and tone. They could then indicate in a reply comment that they have made the changes and thank the reviewer for their feedback. If the author disagrees with the feedback, they could explain their reasoning and propose an alternative solution.

Here are some examples of possible responses to this comment:

* "Thanks for the feedback, @SeineSailor! I've reviewed the text and made some changes to improve the grammar and tone. Please let me know if there's anything else I can do to improve it."
* "I appreciate your feedback, @SeineSailor. I disagree with the suggested change to the tone because [reasoning]. Instead, I propose [alternative solution]. Please let me know if this works for you."
* "Hi @SeineSailor, I've looked over the text and I think it's already clear and professional. However, I'm not a native English speaker, so if you see anything that could be improved, please let me know. Thanks!"